src-vue/src/views/ThunderstoreModsView.vue

```diff
@@ -0,0 +1,214 @@
+<template>
+    <div style="height: calc(100% - var(--fc-menu_height))">
+        <div v-if="mods.length === 0" class="fc__changelog__container">
+            <el-progress :show-text="false" :percentage="50" :indeterminate="true" />
+        </div>
+        <el-scrollbar v-else class="container">
+            <!-- Search filters -->
+            <div class="filter_container">
+                <el-input v-model="input" placeholder="Search" clearable @input="onFilterTextChange" />
+            </div>
+
+            <!-- Message displayed if no mod matched searched words -->
+            <div v-if="filteredMods.length === 0 && input.length !== 0" class="noModFound">
+                No matching mod has been found.<br/>
+                Try another search!
+            </div>
+
+            <!-- Mod cards -->
+            <el-card v-for="mod of modsList" v-bind:key="mod.name" :body-style="{ padding: '0px' }">
+                <img
+                    :src="mod.versions[0].icon"
+                    class="image"
+                />
+                <div style="padding: 0px 10px 10px;">
+                    <span class="statContainer">
+                        <el-icon class="no-inherit">
+                            <Download />
+                        </el-icon>
+                        {{ modDownloadsCount(mod) }}
+                    </span>
+
+                    <span class="statContainer">
+                        {{ mod.rating_score }}
+                        <el-icon class="no-inherit">
+                            <Star />
+                        </el-icon>
+                    </span>
+                    <br/>
+
+                    <div class="name hide-text-overflow">{{ mod.name }}</div>
+                    <div class="author hide-text-overflow">by {{ mod.owner }}</div>
+                    <div class="desc">
+                        {{ mod.versions[0].description }}
+                    </div>
+                    <span style="display: flex">
+                        <el-button type="primary" style="flex: 6">
+                            {{ getModButtonText(mod) }}
+                        </el-button>
+                        <el-button link type="info" class="infoBtn" @click="openURL(mod.package_url)">
+                            <el-icon>
+                                <InfoFilled />
+                            </el-icon>
+                        </el-button>
+                    </span>
+                </div>
+            </el-card>
+        </el-scrollbar>
+    </div>
+</template>
+
+<script lang="ts">
+import {defineComponent} from 'vue';
+import {ThunderstoreMod} from "../utils/thunderstore/ThunderstoreMod";
+import { shell } from '@tauri-apps/api';
+import { ThunderstoreModVersion } from '../utils/thunderstore/ThunderstoreModVersion';
+
+export default defineComponent({
+    name: "ThunderstoreModsView",
+    /**
+     * TODO Mods should be stored in front store
+     * TODO Fetch mods with backend?
```

Comment:
For completion's sake:

As mentioned in DMs, I'm totally fine with doing it in front-end for now and moving it to backend later. As such the TODO comments should stay preserved here. 

So basically this code comment can be resolved as is and acts as a free-dopamine-button :P

Reply:
re: 0f6e78246ca41b125785a24664d49ae0121b1909

lol, my comment came in too late xD

Reply:
I removed this while you were typing your comment :)